backend/server/src/main/java/ai/verta/modeldb/DAOSet.java

```diff
@@ -49,7 +47,6 @@ public class DAOSet {
   @JsonProperty private LineageDAO lineageDAO;
   @JsonProperty private MetadataDAO metadataDAO;
   @JsonProperty private RepositoryDAO repositoryDAO;
-  @JsonProperty private FutureEventDAO futureEventDAO;
```

Comment:
not needed for this PR, but I also don't know why we have json-related annotations on this class. I'm 100% sure we never try to turn it into JSON anywhere.